modules/mw.cx.init.Translation.js

```
@@ -258,7 +270,16 @@ mw.cx.init.Translation.prototype.attachToDOM = function (veTarget) {
  * @param {mw.cx.SiteMapper} siteMapper
  * @return {Promise}
  */
+
 mw.cx.init.Translation.prototype.fetchSourcePageContent = function (wikiPage, targetLanguage, siteMapper) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)